problems/RF_Plasma.i

```diff
@@ -0,0 +1,357 @@
+#This is the Action version of Lymberopoulos_with_argon_metastables.i
+
+dom0Scale=25.4e-3
+
+[GlobalParams]
+  potential_units = kV
+  use_moles = true
+[]
+
+[Mesh]
+  [./geo]
+    type = FileMeshGenerator
+    file = 'Lymberopoulos.msh'
+  [../]
+  [./left]
+    type = SideSetsFromNormalsGenerator
+    normals = '-1 0 0'
+    new_boundary = 'left'
+    input = geo
+  [../]
+  [./right]
+    type = SideSetsFromNormalsGenerator
+    normals = '1 0 0'
+    new_boundary = 'right'
+    input = left
+  [../]
+[]
+
+[Problem]
+  type = FEProblem
+[]
+
+[DriftDiffusionAction]
+  [./Plasma]
+    electrons = em
+    charged_particle = Ar+
+    Neutrals = Ar*
+    potential = potential
+    Is_potential_unique = true
+    mean_energy = mean_en
+    position_units = ${dom0Scale}
+    Additional_Outputs = 'ElectronTemperature Current EField'
+  [../]
+[]
```

Comment:
Is it possible to write multiple blocks here for a single domain? e.g. [./plasma_nitrogen], [./plasma_oxygen], etc. for an air plasma, all in the same block). I also have a drift diffusion action that I've been using (yours looks better so I'll be replacing it when this PR goes through), but I've been working with input files that have ~30 species and I found that it's much easier for a user readability standpoint to be able to split up those variables by species instead of having "charged_particle" be a list of 30 species. It's easier to read that way.

Not a necessary thing, just curious.

Reply:
Not currently, since the action adds the potential "diffusion" to each block it is used in. So calling it in multiple times for in one block will cause that block to have multiple potential diffusion kernels. I can add an bool statement like "add_potential_diffusion", which should fix that and then you can add the action multiple times in one block.